services/protocols/MS-Graph-API.js

```diff
@@ -214,37 +216,34 @@ async function _generateMsGraphApiEmailProperties(email) {
     body.content = "";
   }
 
-  const toRecipients = [];
-  for (const address of email.emailTo.split(",")) {
-    toRecipients.push({
-      emailAddress: {
-        address: address,
-      },
-    });
-  }
-  const ccRecipients = [];
-  for (const address of email.emailCc.split(",")) {
-    toRecipients.push({
-      emailAddress: {
-        address: address,
-      },
-    });
-  }
-  const bccRecipients = [];
-  for (const address of email.emailBcc.split(",")) {
-    bccRecipients.push({
+  const emailToMSApiRecipient = (email) => {
+    if (email === undefined || email === null) {
+      return null;
+    }
+    return {
       emailAddress: {
-        address: address,
+        address: email,
       },
-    });
-  }
+    };
+  };
+  const splitEmailString = (emails) => {
+    if (emails === undefined || emails === null) {
+      return null;
+    }
+    const array = [];
+    for (const email of emails.split(",")) {
+      array.push(emailToMSApiRecipient(email));
+    }
+    return array;
+  };
 
   const mailProperties = {
+    toRecipients: splitEmailString(email.emailTo),
+    ccRecipients: splitEmailString(email.emailCc),
+    bccRecipients: splitEmailString(email.emailBcc),
+    replyTo: splitEmailString(email.replyTo),
```

Comment:
If I interpret the code correctly, if no `reply-to` is specified in the triplestore, this property wil contain
```
replyTo: {
    emailAdress: {
       address: ''
   }
}
```

I think it's more clear to provide `null` or leave out the property in that case. Or did you test whether providing an empty address doesn't overrule the default reply-to behaviour?  (Same comment holds for `to`, `cc` and `bcc` but there is no default behaviour which you may overrule by providing empty strings there)